File: altigo_web/src/core/api/endpoints.ts
export const endpoints = {
  register: '/auth/register',
  login: '/auth/login',
  me: '/users/me',

  routes: '/routes',
  routeDetail: (routeId: string) => `/routes/${routeId}`,
  routePhotos: (routeId: string) => `/routes/${routeId}/photos`,

  subscriptionStatus: '/billing/subscription-status',
  subscribe: '/billing/subscribe',
};

File: altigo_app/lib/core/network/endpoints.dart
/// REST endpoints exposed by the AltiGo Node.js/Express backend.
class Endpoints {
  Endpoints._();

  static const register = '/auth/register';
  static const login = '/auth/login';
  static const me = '/users/me';

  static const routes = '/routes';
  static String routeDetail(String routeId) => '/routes/$routeId';
  static String routePhotos(String routeId) => '/routes/$routeId/photos';

  static const subscriptionStatus = '/billing/subscription-status';
  static const subscribe = '/billing/subscribe';
}
